packages/components/slider/src/composables/use-slider-button.ts

```diff
@@ -76,15 +76,13 @@ export const useSliderButton = (
   const button = ref<HTMLDivElement>()
 
   const currentPosition = computed(() => {
-    return `${
-      ((props.modelValue - min.value) / (max.value - min.value)) * 100
-    }%`
+    return ((props.modelValue - min.value) / (max.value - min.value)) * 100
```

Comment:
The logic here seems to be the same as before, and there should be no need to change it. 🤔

Reply:
> The logic here seems to be the same as before, and there should be no need to change it. 🤔

Before modification, it was a string like 30%, now it is a number like 30.

There's no need to convert 30% to 30 later on at line 116-118 and line 196